src/shared/sign/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./sign";
```

Comment:
to move to v2 as discussed

Reply:
Moved to `v2` in 608a779, will shift it back to `shared` once it has been implemented for `v3`.